src/bidiMapper/domains/context/BrowsingContextImpl.ts

```diff
@@ -704,56 +707,37 @@ export class BrowsingContextImpl {
     await this.#cdpTarget.cdpClient.sendCommand('Page.bringToFront');
   }
 
-  async captureScreenshot(): Promise<BrowsingContext.CaptureScreenshotResult> {
+  async captureScreenshot(
+    params: BrowsingContext.CaptureScreenshotParameters
+  ): Promise<BrowsingContext.CaptureScreenshotResult> {
+    if (!this.isTopLevelContext()) {
+      throw new UnsupportedOperationException(
+        `Non-top-level 'context' is currently not supported`
```

Comment:
Did you mean:

`Non-top-level context ${contextId} is currently not supported`

Otherwise I don't get why `context` is in quotes.

Either way: Printing the ID here would be nicer, IMHO.

Reply:
Because it's referencing the parameter.